src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.css

```diff
@@ -130,6 +135,6 @@
   align-items: flex-start;
 }
 
-.marginBottom {
-  margin-bottom: 24px;
+.marginTop {
```

Comment:
We try not to use class names like these. Please, provide a more meaningful name (e.g. container of sorts).